prettier.config.js

```diff
@@ -1,5 +1,14 @@
+'use strict';
+
 module.exports = {
-  parser: 'typescript',
+  arrowParens: 'avoid',
+  bracketSpacing: true,
+  parser: 'babylon',
+  jsxBracketSameLine: true,
+  proseWrap: 'preserve',
+  semi: true,
   singleQuote: true,
-  trailingComma: 'all',
```

Comment:
man github does some weird things with diffs sometimes..
It was changed to `es5` so that the codebase kept compatibility with node v6.
but if we're dropping that then we can keep it at `all`!